tracks/elm/exercises/bob/mentoring.md

```diff
@@ -0,0 +1,41 @@
+This is a good exercise to talk about bringing domain concepts (such as `isYelling`, `isQuestion` and so on) in to the code, as it is quite easy and most students will do it automatically.
+
+This exercise introduces some string handling functions, such as `String.trim`, `String.endsWith`, `String.any`, `String.isEmpty`, `Char.isAlpa` and `Char.isUpper`
+
+There are some good and sensible opportunities to use point free style.
+
+There are also some general code quality things to discuss, such as keep related logic together, and ordering the code by importance. Students should strive to communicate what the code does in as few lines as possible (preferably just the main function and argument names). They show also strive to communicate how it does it in as few lines as possible, preferably just the main function body.
+
+### Reasonable solutions
+
+There are two main styles of solution.
+
+One involves if statements and helper functions to calculate `isYelling` and suchlike. This is a good solution, but has the minor drawback that you either have to repeat some logic in the conditional statements (which is preferred), or use nested conditional statements (which should be avoided).
+
+The other involves pattern matching with helper functions to calculate `isYelling` and suchlike. This often reads slightly better, but uses a few more lines of code.
+
+Either solution is acceptable, and generally as long as the tests pass it is ok to approve any solution which isn't truly terrible. Comments should be added for any suggested improvements or talking points.
+
+### Common suggestions
+
+`isSilence` is easy to calculate, and is probably best as `String.trim remark |> String.empty remark` or `String.trim >> String.empty` (point free style)
```

Comment:
```suggestion
`isSilence` is easy to calculate, and is probably best as `String.trim remark |> String.isEmpty` or `String.trim >> String.isEmpty` (point free style)
```
Isn't it `String.isEmpty` in both cases, and without the extra argument in case of `|>`?

Reply:
Sharp eyes! Yes, I've corrected this